modules/gsp-cluster/variables.tf

```diff
@@ -113,3 +112,29 @@ variable "codecommit_init_role_arn" {
   type    = "string"
   default = ""
 }
+
+variable "network_id" {
+  type = "string"
+}
+
+variable "private_subnet_ids" {
+  type = "list"
+}
+
+variable "public_subnet_ids" {
+  type = "list"
+}
+
+variable "nat_gateway_public_ips" {
+  type = "list"
+}
+
+variable "cert_pem" {
+  description = "Sealed secrets cert"
+  type        = "string"
+}
+
+variable "private_key_pem" {
```

Comment:
Could we rename this to something like `sealed_secrets_key_pem` so we don't need to rely on the description to determine its use?